components/combobox/combobox.jsx

```diff
@@ -340,8 +354,20 @@ class Combobox extends React.Component {
 		if (this.props.errorText) {
 			this.generatedErrorId = shortid.generate();
 		}
+
+		if (this.props.isOpen !== this.state.isOpen) {
```

Comment:
@shleewhite Could you tell me more about this conditional? At this point, `this.state.isOpen` is always going to be false I think.

Reply:
This was for the snapshot test. If I didn't have this, the popover wouldnt be open.

Reply:
Would `(this.props.isOpen)` be sufficient then?

Reply:
You're right! I'll change that now.